src/ctapipe/irf/visualisation.py

```diff
@@ -0,0 +1,224 @@
+import astropy.units as u
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.visualization import quantity_support
+from matplotlib.colors import LogNorm
+from pyirf.binning import join_bin_lo_hi
+
+from .vis_utils import (
+    find_columnwise_stats,
+    get_2d_hist_from_table,
+    get_bin_centers,
+    get_x_bin_values_with_rebinning,
+)
+
+quantity_support()
```

Comment:
Please don't call globally mutating functions in ctapipe on import.

Use as a context manager around the plots where needed.